daemon/task_handler.rs

```diff
@@ -741,6 +741,15 @@ impl TaskHandler {
         parameters.insert("command", task.command.clone());
         parameters.insert("path", task.path.clone());
         parameters.insert("result", task.result.clone().unwrap().to_string());
+
+        let print_time = |time: Option<DateTime<Local>>| {
+            time.map(|time| time.timestamp().to_string())
+                .unwrap_or_else(String::new)
```

Comment:
Hmm.
I'm not sure if a timestamp is the best solution.
It might be a good solution to let the user decide how they want to format stuff.
It's pretty unergonomic, though.

Let me think about this for a little bit :D

Reply:
I thought it would be the most versatile, you can convert it with one inline call to `date` after all. I do agree though that this is something that should be carefully decided because changing after the fact would break every single callback hook that uses it.

Reply:
For what it's worth it's also easier to calculate the runtime this way, since you can just subtract the timestamps instead of having to convert both dates to timestamps first.

Reply:
Ok. After a good night I agree with you :D
Let's use timestamps.